pkg/kubectl/cmd/apply.go

```diff
@@ -128,6 +128,10 @@ func NewCmdApply(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddRecordFlag(cmd)
 	cmdutil.AddInclude3rdPartyFlags(cmd)
+
+	// apply subcommands
+	cmd.AddCommand(NewCmdApplyView(f, out))
```

Comment:
We are going to want to add a `last-applied` subcommand.  And then add a `view` subcommand to that.  It will make the `-h` work.